src/test/java/callhub/connect/controllers/FileControllerTests.java

```diff
@@ -120,4 +125,58 @@ void testFindDocumentByID_InvalidID() throws Exception {
         RequestBuilder request = get("/files/12345678");
         mockMvc.perform(request).andExpect(status().isBadRequest()).andExpect(content().string("File could not be found."));
     }
+
+    @Test
+    void testPDFLinkValidSession() throws Exception {
+        String currentDirectory = System.getProperty("user.dir");
+        String pathName = currentDirectory + "/src/main/java/callhub/connect/pdfs/fall2023.pdf";
```

Comment:
Can we remove this file and create a blank pdf in the test setup for testing instead?